geode-core/src/main/java/org/apache/geode/internal/cache/control/PartitionRebalanceDetailsImpl.java

```diff
@@ -48,6 +48,27 @@
   private long time;
   private int numOfMembers;
 
+  @Override
+  public String toString() {
```

Comment:
I have the same comment as those in the pull request you closed #6839.

Reply:
Actually, I have addressed all of your concerns except the toString and numOfMembers. As I mention (after your comment) I don't think numOfMembers is required for my implementation, so I am hesitant to do anything with it. I will update the toString though.